.circleci/config.yml

```diff
@@ -100,22 +100,24 @@ jobs:
 
   cross-compile:
     executor: golang
+    parallelism: 4
     steps:
       - attach_to_workspace
       - run:
           name: Build collector for all archs
-          command: make binaries-all-sys
+          command: grep ^binaries-all-sys Makefile|fmt -w 1|tail -n +2|circleci tests split|xargs make
       - persist_to_workspace:
           root: ~/
           paths: project/bin
 
   loadtest:
     executor: golang
+    parallelism: 10
     steps:
       - attach_to_workspace
       - run:
           name: Loadtest
-          command: make -C testbed runtests
+          command: TEST_ARGS="-test.run=$(make -s -C testbed listtests | circleci tests split|xargs echo|sed 's/ /|/g')" make -C testbed runtests
```

Comment:
This will break the TESTRESULTS.md artifact in various small ones but it is a worthy price for the faster builds. Could you add one step after all tests to stitch them together? Preferably as a separate PR.

Reply:
sure, can try to.